src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/MinimalGenotypingEngine.java

```diff
@@ -0,0 +1,54 @@
+package org.broadinstitute.hellbender.tools.walkers.genotyper;
+
+import htsjdk.variant.variantcontext.Allele;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.walkers.genotyper.afcalc.AFCalculatorProvider;
+import org.broadinstitute.hellbender.utils.genotyper.SampleList;
+
+
+/**
+ * A stripped-down version of the former UnifiedGenotyper's genotyping strategy implementation,
+ * used only by the HaplotypeCaller for its isActive() determination.
+ */
```

Comment:
make class final and document that it is not supposed to be used for actual genotyping. Maybe it should become a package-private class or even private inside of HaplotypeCallerEngine?